web/src/compose_actions.ts

```diff
@@ -203,83 +248,83 @@ export function start(opts) {
     compose_banner.clear_message_sent_banners();
     expand_compose_box();
 
-    opts = fill_in_opts_from_current_narrowed_view(opts);
-    const is_clear_topic_button_triggered = opts.trigger === "clear topic button";
+    const filled_opts = fill_in_opts_from_current_narrowed_view(opts);
+    const is_clear_topic_button_triggered = filled_opts.trigger === "clear topic button";
```

Comment:
A better pattern for this may be to do `opts = fill_in_opts_from_current_narrowed_view(raw_opts);`, rather than referencing a new `fill_opts` all through the function, which is a lot of churn and a bit less readable.

Reply:
I like that, changed to that.